.github/workflows/lint.yml

```diff
@@ -11,6 +11,14 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/python-setup
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+      - name: Install prettier
+        run: npm install
+      - name: Run Prettier Check
+        run: npx prettier --check "**/*.{yml,yaml,md}"
```

Comment:
Let's separate these linter checks from the python linter checks. Can we run this in parallel?